docs/donations.md

```diff
@@ -0,0 +1,13 @@
+# Contribute
+
+We at T2 Linux, currently do not accept financial donations. Though, some of our contributors/maintainers do accept donations at their own level. You may refer to the [Support our maintainers](https://wiki.t2linux.org/donations/#support-our-maintainers) section of this page to get the links of the same.
```

Comment:
I don't think a `,` is grammatically correct here. Also perhaps this could be phrased a little differently, how about "The T2 project itself currently does not accept finational donations. However, some of the contributors and maintainers of the project to accept donations on their own."?

Reply:
db0f6f2